src/routes/local.ts

```diff
@@ -1,13 +1,46 @@
 import express from 'express'
 import n2yo from './../api/n2yo/index.js'
+import SatelliteUtilities from '../utils/satelliteUtils'
+import _ from 'lodash'
 
 const router = express.Router()
+const propsToPick = ['_id', 'officialName', 'perigee', 'apogee', 'operator', 'operatorCountry', 'purpose']
 
 router.get('/two-line-elements/:id', (req, res) => {
-    if (req.params.id == undefined) return res.sendStatus(400)
+    if (req.params.id === undefined) return res.sendStatus(400)
     n2yo.getTwoLineElements(req.params.id)
         .then(result => res.json(result))
         .catch(() => res.sendStatus(400))
 })
 
+router.get('/satellite', (req, res) => {
```

Comment:
This is a GET route, but your handler reads from `req.body`. GET requests have no body [0]. You probably meant to read query parameters. You can use `req.query` for that.

[0] GET requests can technically be sent with body, but browsers typically don't allow it.